react/features/filmstrip/components/native/DominantSpeakerIndicator.js

```diff
@@ -1,27 +1,24 @@
-import React, { Component } from 'react';
-import { View } from 'react-native';
+// @flow
 
-import { Icon } from '../../../base/font-icons';
+import React, { Component } from 'react';
 
-import styles from './styles';
+import BaseIndicator from './BaseIndicator';
 
 /**
  * Thumbnail badge showing that the participant is the dominant speaker in
  * the conference.
  */
-export default class DominantSpeakerIndicator extends Component {
+export default class DominantSpeakerIndicator extends Component<{}> {
     /**
-     * Implements React's {@link Component#render()}.
+     * Renders the platform specific indicator element.
      *
      * @inheritdoc
      */
-    render() {
+    _renderIndicator() {
```

Comment:
This needs to be called render()

Reply:
Sh*t, this is a remainder of a bigger refactor... thanks for catching it!